app/scripts/modules/core/src/pipeline/config/templates/v2/configurePipelineTemplateModalV2.controller.spec.ts

```diff
@@ -0,0 +1,306 @@
+import { mock, IScope, IQService } from 'angular';
```

Comment:
These tests have been brought over from the V1 controller and modified for the MPTv2 controller

Reply:
Btw thanks for all these really helpful comments, it would have been much more difficult to review otherwise without any context around this code :)

Reply:
glad it is helpful 👍